src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 198px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+    background-size: contain;
+  }
+
+  &__item {
+    font-size: 12px;
+    text-align: left;
+    &-name {
+      color: $main-color;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    &-product-code{
+      font-size: 10px;
+      align-self: flex-start;
+      color: $secondary-color;
+      line-height: 14px;
+      margin-top: 4px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    gap: 80px;
```

Comment:
no need to define such a gap, you have space between, so it should adjust space between elements